### ISSUE TITLE
Dependency issue with TW v3.0.0-alpha.1

### ISSUE THREAD

**Issue by sinyayadynya** [tailwindlabs/tailwindcss-aspect-ratio#24]
### What version of @tailwindcss/aspect-ratio are you using?

v0.2.1

### What version of Node.js are you using?

v14.16.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

https://github.com/sinyayadynya/melon

### Describe your issue

I uninstalled tailwindcss-aspect-ratio before upgrading to TailwindCSS v3.0.0-alpha.1 (thought if was fully native), and then tried to reinstall it, but got this error messages: 

```
npm ERR! Could not resolve dependency:
npm ERR! peer tailwindcss@">=2.0.0" from @tailwindcss/aspect-ratio@0.2.1
```

**Comment by adamwathan** [comment]
Hey, thanks for reporting! Should be fixed in v0.2.2 👍🏻 

```
npm install -D @tailwindcss/aspect-ratio@latest
```

**Comment by dionysiusmarquis** [comment]
I still get an error while trying to audit:

```sh
✗ npm audit fix
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @tailwindcss/aspect-ratio@0.3.0
npm ERR! Found: tailwindcss@2.2.16
npm ERR! node_modules/tailwindcss
npm ERR!   dev tailwindcss@"^2.2.16" from the root project
npm ERR!   peer tailwindcss@">=2.0.0 || >=3.0.0-alpha.1" from @tailwindcss/aspect-ratio@0.3.0
npm ERR!   node_modules/@tailwindcss/aspect-ratio
npm ERR!     dev @tailwindcss/aspect-ratio@"^0.3.0" from the root project
npm ERR!   1 more (@tailwindcss/line-clamp)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer tailwindcss@">=2.0.0 || >=3.0.0-alpha.1" from @tailwindcss/aspect-ratio@0.3.0
npm ERR! node_modules/@tailwindcss/aspect-ratio
npm ERR!   dev @tailwindcss/aspect-ratio@"^0.3.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: tailwindcss@3.0.0-alpha.1
npm ERR! node_modules/tailwindcss
npm ERR!   peer tailwindcss@">=2.0.0 || >=3.0.0-alpha.1" from @tailwindcss/aspect-ratio@0.3.0
npm ERR!   node_modules/@tailwindcss/aspect-ratio
npm ERR!     dev @tailwindcss/aspect-ratio@"^0.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
